aea/runtime.py

```diff
@@ -389,12 +389,8 @@ async def _start_agent_loop(self) -> None:
         )
 
         self._state.set(RuntimeStates.running)
-        try:
-            await self.agent_loop.wait_completed()
-        except asyncio.CancelledError:
-            self.agent_loop.stop()
-            await self.agent_loop.wait_completed()
-            raise
+        # loop will be stopped in wait completed
+        await self.agent_loop.wait_completed()
```

Comment:
Can you point me to the code?

Reply:
**https://github.com/valory-xyz/open-aea/blob/main/aea/agent_loop.py#L124**

so i captures cancelled event and stops agent loop.
checked it.